docs/faq.md

```diff
@@ -65,9 +65,10 @@ OSS-Fuzz is a production instance of ClusterFuzz, plus the code living in
 
 ## Why do you use a [different issue tracker](https://bugs.chromium.org/p/oss-fuzz/issues/list) for reporting bugs in OSS projects?
 
-Security access control is important for the kind of issues that OSS-Fuzz detects.
-We will reconsider the GitHub issue tracker once the
-[access control feature](https://github.com/isaacs/github/issues/37) is available.
+Security access control is important for the kind of issues that OSS-Fuzz detects,
+hence why by default issues are only opened on the OSS-Fuzz tracker.
+You can opt-in to have them on Github as well by adding the `file_github_issue`
```

Comment:
Can you add something here about how it's just for the purpose of visibility and that the actual details can be found by following the link to monorail?